cmd/cli/job/list_test.go

```diff
@@ -0,0 +1,120 @@
+//go:build integration || !unit
+
+package job_test
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	cmdtesting "github.com/bacalhau-project/bacalhau/cmd/testing"
+	"github.com/bacalhau-project/bacalhau/pkg/docker"
+	"github.com/bacalhau-project/bacalhau/pkg/system"
+)
+
+// In order for 'go test' to run this suite, we need to create
+// a normal test function and pass our suite to suite.Run
+func TestListSuite(t *testing.T) {
+	suite.Run(t, new(ListSuite))
+}
+
+// Define the suite, and absorb the built-in basic suite
+// functionality from testify - including a T() method which
+// returns the current testing context
+type ListSuite struct {
+	cmdtesting.BaseSuite
+}
+
+// Before each test
+func (s *ListSuite) SetupTest() {
+	docker.MustHaveDocker(s.T())
+	s.BaseSuite.SetupTest()
+}
+
+func (s *ListSuite) TestJobList() {
+	// Submit a job
+	args := []string{
+		"docker", "run",
+		"--wait",
+		"alpine", "echo", "hello world",
+	}
+	_, out, err := s.ExecuteTestCobraCommand(args...)
+	require.NoError(s.T(), err, "Error submitting job")
```

Comment:
Minor comment for all assertions:
Since you already created the Test suite, you can use `s.Require().NoError...` , `s.Require().Equal...` .